pandas/io/pytables.py

```diff
@@ -1133,6 +1133,10 @@ def _create_storer(self, group, format=None, value=None, append=False,
                        **kwargs):
         """ return a suitable class to operate """
 
+        if isinstance(value, Series) and any(np.iscomplex(value)) and format=='table':
```

Comment:
Needs to go here: https://github.com/pydata/pandas/blob/master/pandas/io/pytables.py#L1785